README.md

```diff
@@ -6,6 +6,20 @@ earthquake and possibly other hazardous situations. It is currently hosted at
 
 ![Screenshot](doc/img/screenshot.png)
 
+## Json API
+
+### /api/ads
+
+Returns ads with pagination details in headers (`Link`, `Per-Page`, `Total`). This follows the proposed [RFC-8288](https://tools.ietf.org/html/rfc8288) standard for Web linking.
+
+Available params (all params are optional):
+
+| Param name | Allowed values | Default | Example |
+| page | Integer gt 0 | 1 | https://www.potres-petrinja.hr/api/ads?page=2 |
+| per_page | Integer 20 <= i <= 100 | 20 | https://www.potres-petrinja.hr/api/ads?per_page=50 |
+| kind | `supply` or `demand` | - | https://www.potres-petrinja.hr/api/ads?kind=demand |
+| category | `accomodation`, `transport`, `repairs`, `medical_assistance`, `other`, `building_material`, `kids`, `diet_and_hygiene`, `furniture_and_household`, `clothes_and_shoes` | - | https://www.potres-petrinja.hr/api/ads?category=accomodation |
```

Comment:
Seems like you need to mark the separator between table header and body for it to render in gh flavored markdown as table

```suggestion
| Param name | Allowed values | Default | Example |
| --- | --- | --- | --- |
| page | Integer gt 0 | 1 | https://www.potres-petrinja.hr/api/ads?page=2 |
| per_page | Integer 20 <= i <= 100 | 20 | https://www.potres-petrinja.hr/api/ads?per_page=50 |
| kind | `supply` or `demand` | - | https://www.potres-petrinja.hr/api/ads?kind=demand |
| category | `accomodation`, `transport`, `repairs`, `medical_assistance`, `other`, `building_material`, `kids`, `diet_and_hygiene`, `furniture_and_household`, `clothes_and_shoes` | - | https://www.potres-petrinja.hr/api/ads?category=accomodation |
```


https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/organizing-information-with-tables